examples/yml-subgraph-http-test/router.yaml

```diff
@@ -0,0 +1,26 @@
+supergraph:
+  listen: 127.0.0.1:4000
+  introspection: true
+sandbox:
+  enabled: true
+homepage:
+  enabled: false
+include_subgraph_errors:
+  all: true # Propagate errors from all subraphs
+
+plugins:
+  # this plugin has one piece of configuration,
+  # the name of the entity you'd like to say hello to
+  example.echo_co_processor:
+    port: 8080
+
+  experimental.external:
+    url: http://127.0.0.1:8080 # mandatory URL which is the address of the co-processor
+    timeout: 2s # optional timeout (2 seconds in this example). If not set, defaults to 1 second
+    stages:
+      subgraph: # router or subgraph
+        request: # What data should we transmit to the co-processor from the router request?
+          headers: true # All of these data content attributes are optional and false by default.
+          context: true
+          body: true
+          sdl: false
```

Comment:
i'm not sure how useful the SDL is. subgraph requests with _entities aren't valid against the supergraph.

Reply:
true, i can remove it!

Reply:
done!